typescript/schemas/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
Maybe extend `@foxglove/tsconfig/base` unless there is a strong reason to make the package dependent on its parent dir?

Reply:
done